chapters/bn/_toctree.yml

```diff
@@ -0,0 +1,173 @@
+- title: 0. সেটআপ
+  sections:
+  - local: chapter0/1
+    title: ভূমিকা
+
+- title: 1. ট্রান্সফরমার মডেল
```

Comment:
Thanks for translating the whole Table of Contents, but we should only include the sections that have a corresponding set of translated MDX files. In this case, we should remove everything from this line downwards

Reply:
@lewtun Thanks for the quick review. Got it 👍🏾 
I just changed accordingly. Hope this will fix the issue.

Reply:
Oh sorry, what I meant is that `_toctree.yml` should only include this:

```
- title: 0. সেটআপ
  sections:
  - local: chapter0/1
    title: ভূমিকা
```

Everything else should be removed until we have the corresponding MDX files translated. Would you mind fixing that? Sorry for the bother!

Reply:
Thank you for iterating - this looks great!